components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/config/CopyFile.java

```diff
@@ -23,7 +23,7 @@
  * annotation.
  */
 public class CopyFile {
-    private String source;
+    private String sourceFile;
```

Comment:
@Rajith90 @hasuniea @AmaliMatharaarachchi 
hmm,
This change depends on how we are going to version the first jballerina based mgw release. This change makes older deployement-config files invalid isn't it?
If we are good to do 4.0.0 i'm ok with this change, if not -1

Reply:
Reverted the change in https://github.com/wso2/product-microgateway/pull/862/commits/bb187e88405824d88734578b6a95503ed078babb, https://github.com/wso2/product-microgateway/pull/862/commits/013f9019e5a9ed2f874b9ef63696e554486d1fcb